README.md

```diff
@@ -34,3 +34,11 @@ chmod a+x ./hibp.pl
 </code>
 
 It runs in a infinite loop until you enter an empty password.
+
+
+## Dependencies
+
+This script depends on the Digest::SHA and the Term::Readkey perl modules.
+On Debian-based systems (Debian, Ubuntu, Mint etc.) they can be installed by
+
+	sudo apt install libdigest-sha-perl libterm-readkey-perl
```

Comment:
Fine. I wrote this on a Mac and had these modules already installed (I don't know if I did this in the past manually or they were preinstalled. 

There is also a perl specific way to install modules:

perl -M CPAN -e shell
install Digest::SHA
install Term::ReadKey

That should work universally. But your suggestions are fine if you want to worry just about your distribution handling updates.